usr/share/rear/verify/COMMVAULT/default/420_login_to_galaxy_and_setup_environment.sh

```diff
@@ -0,0 +1,32 @@
+#
+# Commvault requires a logon to the backup system that is independent from the system
+# logon. The logon stores a session file on the system (/opt/commvault/qsession.0) so that
+# a session might exist already.
+
+# set path to Commvault
+export PATH=$PATH:/opt/commvault/Base
```

Comment:
At least offhandedly it looks dangerous to modify the PATH.
Doesn't this modify the PATH in all subsequent scripts?
If yes, is it sufficiently fail-safe to run all
subsequent scripts with a modified PATH?

Reply:
Yes, agree. But worth to note that nearly all of the stuff that you noticed was copied over from GALAXY10...

I'll try to clean up what I can

Reply:
Oh yes, I know we have much old code.
I didn't dare to overhaul code for backup tools that I don't have
(so I could test nothing at all) except perhaps some small fixes
or cleanup here and there but no substantial overhaul.
E.g. I cannot clean up `export PATH=...` because I cannot
imagine where such a setting might be (implicitly) needed
in all the subsequent scripts, so I leave such code as is.

Reply:
PATH moved to system configuration